scripts/moveFiles.bash

```diff
@@ -6,6 +6,10 @@ pushd ../publicgoods-website/registry && \
     git add static/* && \
     cp ../../publicgoods-scripts/packages/registry/build/index.html . && \
 popd
+pushd ../publicgoods-website/eligibility && \
+    cp -R ../../publicgoods-website/wp-content/themes/dpga/font/. ../../publicgoods-scripts/packages/eligibility/src/font/ && \
+popd
+pushd packages/eligibility && npm install && npm run build && popd
```

Comment:
The `npm build` does not belong here, as it is included in another step. Or if we want to move things around, then we should be consistent and include the build for the `registry`, which is not included in the preceding section.

Reply:
Okay, I remember now. Running line 12 here was a crutch.
It is because I didn't want to split `moveFiles.bash` into two files just to run line 12 before continuing from line 12 to the end of the bash file. 
It was my attempt to keep only one file.
What do you think?

Reply:
I would refer you back to [this file](https://github.com/unicef/publicgoods-candidates/blob/main/scripts/push.bash), which is the one that [runs the CI](https://github.com/unicef/publicgoods-candidates/blob/9582750b80379535fb115a6d813375de40dcc2e0/.github/workflows/auto.yml#L23) of the [unicef/publicgoods-candidates](https://github.com/unicef/publicgoods-candidates).

Notice how on that file, it runs the registry build on L13, the eligibility build on L16, and **then** it runs moveFiles.bash to move the output of those builds to their corresponding locations in the website folders. So if you are now including the eligibility build inside moveFiles.bash, you are replicating work already done; that's why I was suggesting to include the eligibility build in `.github/workflows/auto.yml` to mirror the structure of the [unicef/publicgoods-candidates](https://github.com/unicef/publicgoods-candidates) and keep it modular to more easily spot where it may fail.

Reply:
I understand a bit more, I'll try this out.